### PR TITLE
Release elastic-job-cloud 2.1.4

### PR DESCRIPTION
**Pull request by haocao**
Initial release of Elastic Job Cloud

### CODE REVIEW

**Comment by OhRobin** [comment]
Please provide an example here: https://github.com/dcos/examples

**Comment by haocao** [comment]
Already provided the example here: https://github.com/dcos/examples/pull/181, please help to check.

**Comment by ryadav88** [comment]
@haocao the package doesn't install. Please validate. 


```[ERROR] 2017-06-25 02:27:19,263 --main-- [org.apache.curator.framework.imps.CuratorFrameworkImpl] Background exception was not retry-able or retry gave up 
java.net.UnknownHostException: mesos.master: Name or service not known
	at java.net.Inet4AddressImpl.lookupAllHostAddr(Native Method) ~[na:1.8.0_121]
	at java.net.InetAddress$2.lookupAllHostAddr(InetAddress.java:928) ~[na:1.8.0_121]
	at java.net.InetAddress.getAddressesFromNameService(InetAddress.java:1323) ~[na:1.8.0_121]
	at java.net.InetAddress.getAllByName0(InetAddress.java:1276) ~[na:1.8.0_121]
	at java.net.InetAddress.getAllByName(InetAddress.java:1192) ~[na:1.8.0_121]
	at java.net.InetAddress.getAllByName(InetAddress.java:1126) ~[na:1.8.0_121]
	at org.apache.zookeeper.client.StaticHostProvider.<init>(StaticHostProvider.java:61) ~[zookeeper-3.4.6.jar:3.4.6-1569965]
	at org.apache.zookeeper.ZooKeeper.<init>(ZooKeeper.java:445) ~[zookeeper-3.4.6.jar:3.4.6-1569965]
	at org.apache.curator.utils.DefaultZookeeperFactory.newZooKeeper(DefaultZookeeperFactory.java:29) ~[curator-client-2.10.0.jar:na]
	at org.apache.curator.framework.imps.CuratorFrameworkImpl$2.newZooKeeper(CuratorFrameworkImpl.java:150) ~[curator-framework-2.10.0.jar:na]
	at org.apache.curator.HandleHolder$1.getZooKeeper(HandleHolder.java:94) ~[curator-client-2.10.0.jar:na]
	at org.apache.curator.HandleHolder.getZooKeeper(HandleHolder.java:55) ~[curator-client-2.10.0.jar:na]
	at org.apache.curator.ConnectionState.reset(ConnectionState.java:218) ~[curator-client-2.10.0.jar:na]
	at org.apache.curator.ConnectionState.start(ConnectionState.java:103) ~[curator-client-2.10.0.jar:na]
	at org.apache.curator.CuratorZookeeperClient.start(CuratorZookeeperClient.java:190) ~[curator-client-2.10.0.jar:na]
	at org.apache.curator.framework.imps.CuratorFrameworkImpl.start(CuratorFrameworkImpl.java:259) ~[curator-framework-2.10.0.jar:na]
	at com.dangdang.ddframe.job.reg.zookeeper.ZookeeperRegistryCenter.init(ZookeeperRegistryCenter.java:98) [elastic-job-common-core-2.1.4.jar:na]
	at com.dangdang.ddframe.job.cloud.scheduler.Bootstrap.main(Bootstrap.java:52) [elastic-job-cloud-scheduler-2.1.4.jar:na]
[ERROR] 2017-06-25 02:27:28,267 --main-- [org.apache.curator.CuratorZookeeperClient]  
java.io.IOException: java.net.UnknownHostException: mesos.master
	at org.apache.curator.ConnectionState.close(ConnectionState.java:119) ~[curator-client-2.10.0.jar:na]
	at org.apache.curator.CuratorZookeeperClient.close(CuratorZookeeperClient.java:203) ~[curator-client-2.10.0.jar:na]
	at org.apache.curator.framework.imps.CuratorFrameworkImpl.close(CuratorFrameworkImpl.java:321) [curator-framework-2.10.0.jar:na]
	at com.dangdang.ddframe.job.reg.zookeeper.ZookeeperRegistryCenter.init(ZookeeperRegistryCenter.java:101) [elastic-job-common-core-2.1.4.jar:na]
	at com.dangdang.ddframe.job.cloud.scheduler.Bootstrap.main(Bootstrap.java:52) [elastic-job-cloud-scheduler-2.1.4.jar:na]
Caused by: java.net.UnknownHostException: mesos.master
	at java.net.InetAddress.getAllByName0(InetAddress.java:1280) ~[na:1.8.0_121]
	at java.net.InetAddress.getAllByName(InetAddress.java:1192) ~[na:1.8.0_121]
	at java.net.InetAddress.getAllByName(InetAddress.java:1126) ~[na:1.8.0_121]
	at org.apache.zookeeper.client.StaticHostProvider.<init>(StaticHostProvider.java:61) ~[zookeeper-3.4.6.jar:3.4.6-1569965]
	at org.apache.zookeeper.ZooKeeper.<init>(ZooKeeper.java:445) ~[zookeeper-3.4.6.jar:3.4.6-1569965]
	at org.apache.curator.utils.DefaultZookeeperFactory.newZooKeeper(DefaultZookeeperFactory.java:29) ~[curator-client-2.10.0.jar:na]
	at org.apache.curator.framework.imps.CuratorFrameworkImpl$2.newZooKeeper(CuratorFrameworkImpl.java:150) ~[curator-framework-2.10.0.jar:na]
	at org.apache.curator.HandleHolder$1.getZooKeeper(HandleHolder.java:94) ~[curator-client-2.10.0.jar:na]
	at org.apache.curator.HandleHolder.internalClose(HandleHolder.java:128) ~[curator-client-2.10.0.jar:na]
	at org.apache.curator.HandleHolder.closeAndClear(HandleHolder.java:71) ~[curator-client-2.10.0.jar:na]
	at org.apache.curator.ConnectionState.close(ConnectionState.java:114) ~[curator-client-2.10.0.jar:na]
	... 4 common frames omitted
Command exited with status 1 (pid: 14)```

**Comment by haocao** [comment]
The package is correct and already installed in our dcos environment. 

`mesos.master` is just a variable, you should replace it with a real Zookeeper server ip address and port , such as `192.168.1.1:2181`.

It's the same meaning as marathon's Zookeeper config:

https://github.com/mesosphere/universe/blob/version-3.x/repo/packages/M/marathon/9/config.json

**Comment by haocao** [comment]
Any feedback about it?

File: repo/packages/E/elastic-job-cloud/0/resource.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+  "assets": {
+    "uris": {
+      "jre-tar-gz": "https://downloads.mesosphere.com/java/jre-8u121-linux-x64.tar.gz",
+      "libmesos-bundle-tar-gz": "https://downloads.mesosphere.com/libmesos-bundle/libmesos-bundle-1.9.0-rc2-1.2.0-rc2-1.tar.gz",
+      "scheduler-zip": "http://dangdangdotcom.github.io/elastic-job/elastic-job-cloud/dist/elastic-job-cloud-scheduler-2.1.4.tar.gz"
+    }
+  },
+  "images": {
+    "icon-small": "https://github.com/dcos/dcos-ui/blob/master/plugins/services/src/img/icon-service-default-small.png?raw=true",
```
Comment (OhRobin) [inline repo/packages/E/elastic-job-cloud/0/resource.json]: Do you wish to change these images to elastic specific?

Comment (haocao) [inline repo/packages/E/elastic-job-cloud/0/resource.json]: Not really, we plan to use our own image later.

**Pull request closed without merging**